osi_hostvehicledata.proto

```diff
@@ -100,6 +100,37 @@ message HostVehicleData
         // Paragraph 42 of the German Road Traffic Admission Regulations (StVZO).
         //
         optional double curb_weight = 1;
+
+        // The operating status of the vehicle.
+        //
+        optional OperatingStatus operating_status = 2;
```

Comment:
Is "Operating status" a commonly used term, it isn't one I'm aware of?  That said, I'm not really in the automotive industry so my not recognising it should be taken for much!

From a quick Google the only result is from a BMW how-to video.

Perhaps `ElectricalState` or `ElectricalOperatingStatus` or similar to make it more obvious?

Once again, don't feel strongly but thought I'd mention.